### PR TITLE
Casehistory/balboa

### PR DESCRIPTION
**Pull request by sgkang**
Incorporate Jean's edits. 

Still require some writing for corresponding changes in figures. 
- [x] Properties (Jean did not put more info. here so not sure what we can do)
- [x] Data (raw ZTEM and Mag data are added)
- [x] Processing (added sections of 2D and 3D inversions)
- [x] Interpretation (section without Balboa outlines): This section is now empty

### CODE REVIEW

File: content/case_histories/balboa/index.rst [hunk 1]
```diff
@@ -1,44 +1,25 @@
...
-======
-
-**The Balboa ZTEM Cu-Mo-Au porphyry discovery at Cobre Panama**
+The Balboa ZTEM Cu-Mo-Au porphyry discovery at Cobre Panama
+===========================================================
 
 .. .. raw:: html
 ..     :file: ../../../underconstruction.html
 
-- **Authors**: :ref:`jlegault`, :ref:`cwijns`, :ref:`cizarra`, and :ref:`gplastow`
```
Comment (lheagy) [inline content/case_histories/balboa/index.rst]: Can you leave this as the refs? unless the names need to be different?

**Comment by lheagy** [comment]
can we generate a ~ chart for the phys props (even relative - eg. high, mod, low) based on the text? do you think that would be sufficient?

**Comment by sgkang** [comment]
We merged processing and interpretation.

File: content/case_histories/balboa/index.rst (hunk 1 above)
Comment (sgkang) [inline content/case_histories/balboa/index.rst]: Not sure exactly what you mean.

Comment (sgkang) [inline content/case_histories/balboa/index.rst]: ah, got it. never mind.

**Pull request merged**